core/src/main/scala/tech/beshu/ror/accesscontrol/logging/LoggingContextFactory.scala

```diff
@@ -0,0 +1,23 @@
+package tech.beshu.ror.accesscontrol.logging
+
+import cats.Show
+import cats.implicits._
+import tech.beshu.ror.accesscontrol.domain.Header
+import tech.beshu.ror.accesscontrol.show.logs._
+
+object LoggingContextFactory {
+  def create(obfuscatedHeaders: Option[ObfuscatedHeaders]): LoggingContext = {
```

Comment:
I think we don't need ObfuscatedHeaders class. I know that there are different approaches according to work with config files:
* parse to intermediate objects and then to core classes
* parse directly to core classes
* [and others]

Your approach is the first one. But in ROR we decided to use 2nd one and we should follow this convention. I know sometimes there are real hard issues to solve using it, but consistency in code base is the most important thing,